sources/packages/web/src/services/http/EducationProgramOfferingApi.ts

```diff
@@ -311,4 +312,28 @@ export class EducationProgramOfferingApi extends HttpBaseClient {
       requestConfig,
     );
   }
+
+  /**
+   * Gets the offering simplified details, not including, for instance,
+   * validations, approvals and extensive data.
+   * Useful to have an overview of the offering details, for instance,
+   * when the user needs need to have quick access to data in order to
+   * support operations like confirmation of enrolment or scholastic
+   * standing requests or offering change request.
+   * @param offeringId offering.
+   * @param options method options:
+   * - `locationId`: location for authorization.
+   * @returns offering details.
+   */
+  async getOfferingSummaryViewById(
+    offeringId: number,
+    options?: {
+      locationId?: number;
+    },
+  ): Promise<EducationProgramOfferingSummaryViewAPIOutDTO> {
+    const url = `education-program-offering/location/${options?.locationId}/offering/${offeringId}/summary-view`;
```

Comment:
adding location to the URL must be based on locationId right?

Reply:
It should, but since there is no other endpoint right now to be called I left it as it is.
I can add it to leave the method looking better even if the alternative endpoint would not work.
I was avoiding adding the locationId as required since the new tasks would change it to optional.

Reply:
Added the second endpoint without the location id for now.